WooCommerce/WooCommerceUITests/Screens/Settings/SettingsScreen.swift

```diff
@@ -32,7 +32,7 @@ final class SettingsScreen: BaseScreen {
 
         // Some localizations have very long "log out" text, which causes the UIAlertView
         // to stack. We need to detect these cases in order to reliably tap the correct button
-        if logOutAlert.buttons.allElementsShareCommonXAxis {
+        if logOutAlert.buttons.allElementsShareCommonAxisX {
```

Comment:
I decided to use "axis x" instead of "x axis" in the library so that if in the future we'll add the same functionality for the y axis, the method names will be homogeneous:

- `allElementsShareCommonAxisX`
- `allElementsShareCommonAxisY`

As opposed to:

- `allElementsShareCommonXAxis`
- `allElementsShareCommonYAxis`